README.md

```diff
@@ -545,6 +546,141 @@ module.export = function migrateData(app) {
 }
 ```
 
+# Force Id
+
```

Comment:
Maybe we can give them a brief description of what `forceId` flag does or link them to our doc if it already exists.